From: Christophe Leroy <christophe.leroy@c-s.fr>
To: Grant Likely <grant.likely@secretlab.ca>, Linus Walleij <linus.walleij@linaro.org>
CC: linux-kernel@vger.kernel.org, stigge@antcom.de
BCC: Christophe Leroy <christophe.leroy@c-s.fr>
Subject: [PATCH] MAX7301 GPIO: Reverting "Do not force SPI speed when using OF Platform"
To: Grant Likely <grant.likely@secretlab.ca>

This patch reverts commit 047b93a35961f7a6561e6f5dcb040738f822b892 which breaks
MAX7301 GPIO driver because that commit was dependant on a rejected patch that
was implementing selection of SPI speed from the Device Tree.

Signed-off-by: Christophe Leroy <christophe.leroy@c-s.fr>

--- linux-3.11-rc6/drivers/gpio/gpio-max7301.c	2013-08-17 21:09:17.000000000 +0200
+++ linux/drivers/gpio/gpio-max7301.c	2013-08-18 06:31:52.000000000 +0200
@@ -56,8 +56,7 @@
 	int ret;
 
 	/* bits_per_word cannot be configured in platform data */
-	if (spi->dev.platform_data)
-		spi->bits_per_word = 16;
+	spi->bits_per_word = 16;
 	ret = spi_setup(spi);
 	if (ret < 0)
 		return ret;
